konig-spreadsheet/src/test/java/io/konig/spreadsheet/WorkbookLoaderTest.java

```diff
@@ -79,26 +80,6 @@
 public class WorkbookLoaderTest {
 	
 
-	@Test
-	public void testInvalidOntologyNamespace() throws Exception {
-
-		InputStream input = getClass().getClassLoader().getResourceAsStream("invalid-ontology-namespace.xlsx");
-		Workbook book = WorkbookFactory.create(input);
-		Graph graph = new MemoryGraph();
-		NamespaceManager nsManager = new MemoryNamespaceManager();
-		graph.setNamespaceManager(nsManager);
-		
-		WorkbookLoader loader = new WorkbookLoader(nsManager);
-		Throwable error = null;
-		try {
-			loader.load(book, graph);
-		} catch (Throwable e) {
-			error = e;
-		}
-		assertTrue(error != null);
-		assertEquals("Namespace must end with '/' or '#' but found: http://schema.org", error.getMessage());
-	}
-	
```

Comment:
This Unit Test Case (testInvalidOntologyNamespace) should NOT be removed.  Please add it back.